src/reporter.ts

```diff
@@ -24,7 +24,7 @@ export interface Reporter {
   onStdErr(chunk: string | Buffer, test?: Test): void;
   onTestEnd(test: Test, result: TestResult): void;
   onTimeout(timeout: number): void;
-  onError(error: TestError, file?: string): void;
+  onError(error: TestError): void;
```

Comment:
This is now a breaking change

Reply:
Updated the description per offline discussion. PTAL.